### DISCUSSION TITLE
I purchased 50gig of space, my local repo is half that size, but PUSH to origin always fails

### DISCUSSION THREAD

**Discussion by RickBlacker** [community/community#46341] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all... I have a pretty large upload.  It's an Unreal engine project.  Around 15 gig. 
I've enabled LFS, but every time I try to upload, it ends up dying on me. 

```
Enumerating objects: 5367, done./279), 644 MB | 741 KB/s
Counting objects: 100% (5367/5367), done.
Delta compression using up to 24 threads
Compressing objects: 100% (4590/4590), done.
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
send-pack: unexpected disconnect while reading sideband packet
Writing objects: 100% (5367/5367), 15.00 GiB | 5.25 MiB/s, done.
Total 5367 (delta 719), reused 5367 (delta 719), pack-reused 0
fatal: the remote end hung up unexpectedly
Everything up-to-date
```

Am I going to have to break my project apart and upload it little by little?

Any help will be greatly appreciated.

**Comment by jge162** [comment]
You can try increasing the buffer size in Git configuration to allow for larger uploads. You can do this by running the command `git config --global http.postBuffer 157286400` (where 157286400 is 150 MB).

If that does not work then yes, you could try breaking the project into smaller parts and uploading them individually.

**Comment by RickBlacker** [comment]
Thanks...  That was one of the things I did try earlier today. Didn't seem to have any impact.

I was just about to try disabling McAfee.  I read that sometimes virus scanners can impact the upload.  I would think though that if that was the case it would have prevented it from the gitgo.  I had been playing with GIT this week and had sucessful uploads but they were a lot smaller in size.  So... I dunno... Probably have to resort to breaking it up into smaller chunks to get it up there.

**Reply by jge162** [reply]
@RickBlacker Disabling your virus scanner is a good step to try if you suspect that it may be interfering with your uploads!!!

**Reply by jge162** [reply]
If that has no impact, then sadly you do have to break up the uploads.

**Reply by RickBlacker** [reply]
I am trying that now.  I started a new push this morning and forgot to turn it off.  And of course it fails after about an hour of uploading.  So, starting again... If this doesn't work, then I'll start breaking my project up into smaller uploads.

**Comment by RickBlacker** [comment]
I have had to resort to breaking this up to smaller chunks to get everything up there.  Initial push is less than 500meg.  Is there a setting up here on the Github website that we are supposed to set on the repository?  Or does the client side, when pushing up to origin take care of everything?

I deleted all GIT stuff from project
I created a new repo
I copied in my GIT-LFS .gitattributes file (absolutely has the file type specified in the error below)
I have installed GIT-LFS already couple days ago
I have bought a 50gig package.

Here is my .gitattributes file contents.


```
*.uasset filter=lfs diff=lfs merge=lfs -text
*.umap filter=lfs diff=lfs merge=lfs -text
*.fbx filter=lfs diff=lfs merge=lfs -text
*.3ds filter=lfs diff=lfs merge=lfs -text
*.psd filter=lfs diff=lfs merge=lfs -text
*.png filter=lfs diff=lfs merge=lfs -text
*.mp3 filter=lfs diff=lfs merge=lfs -text
*.wav filter=lfs diff=lfs merge=lfs -text
*.xcf filter=lfs diff=lfs merge=lfs -text
*.jpg filter=lfs diff=lfs merge=lfs -text
/RawContent/**/* filter=lfs diff=lfs merge=lfs -text

```


Now I'm getting different errors, 
![image](https://user-images.githubusercontent.com/6207302/216788437-6fd15c5d-5ad3-48f4-ba09-dc6e5722ee53.png)
